README.md

```diff
@@ -62,44 +61,41 @@ https://github.com/user-attachments/assets/c9bac609-5b05-417f-b2b2-e57d650d0dd6
 
 ![Export-1724891449817](https://github.com/user-attachments/assets/1f317ae1-6453-4a00-8801-f005ccc7efdb)
 
-   
 ### Built With
-* [![React][React.js]][React-url]
-* [![Electron][Electron.js]][Electron-url]
-* [![Tailwind][TailwindCSS]][Tailwind-url]
-* [![Vite][Vite.js]][Vite-url]
+
+-   [![React][React.js]][React-url]
+-   [![Electron][Electron.js]][Electron-url]
+-   [![Tailwind][TailwindCSS]][Tailwind-url]
+-   [![Vite][Vite.js]][Vite-url]
 
 ## Stay up-to-date
-Onlook officially launched our first version of Onlook on July 08, 2024 and we've shipped a ton since then. Watch releases of this repository to be notified of future updates, and you can follow along with us on [LinkedIn](https://www.linkedin.com/company/onlook-dev) or [Substack](https://onlook.substack.com/) where we write a weekly newsletter. 
 
+Onlook officially launched our first version of Onlook on July 08, 2024 and we've shipped a ton since then. Watch releases of this repository to be notified of future updates, and you can follow along with us on [LinkedIn](https://www.linkedin.com/company/onlook-dev) or [Substack](https://onlook.substack.com/) where we write a weekly newsletter.
 
 <!-- ![Starproject](https://github.com/user-attachments/assets/07742b21-dd98-4be3-b6a6-13d8132af398) -->
 
-
 ## Getting Started
 
 ![image](https://github.com/user-attachments/assets/18b6ad5a-1d5a-4396-af8c-8b85936acf39)
 
 ### Installation
+requirements:
+- [bun.sh](https://bun.sh/docs/installation)
```

Comment:
Is this a hard requirement now? I see it hard-coded in a few places but we can keep it flexible by using npm. Still compatible with bun in that case. What do you think @hitaspdotnet ?